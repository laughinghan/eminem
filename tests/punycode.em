let .punycode (ffi.module.require "punycode")
println(punycode.decode "maana-pta")
println(punycode.decode "--dqo34k")
println(punycode.encode "mañana")
println(punycode.encode "☃-⌘")
