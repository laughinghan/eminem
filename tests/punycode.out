mañana
☃-⌘
maana-pta
--dqo34k
